src/components/IconLink.vue

```diff
@@ -0,0 +1,93 @@
+<script setup lang="ts">
+import type { ComputedRef } from 'vue';
+import { computed, useSlots } from 'vue';
+
+import jiraLogo from '@/assets/jira.svg'
+import gitHubLogo from '@/assets/github.svg'
+import externalLink from '@/assets/externallink.svg'
+import gcpLogo from '@/assets/gcp.png'
+
+const slots = useSlots()
+
+enum LinkTypes {
+    Jira,
+    GitHub,
+    GCP,
+    Internal,
+    External
```

Comment:
feature request: bugzilla

Reply:
More specifically: bugzilla icon: https://bugzilla.mozilla.org/extensions/BMO/web/images/favicon.ico

Reply:
added in https://github.com/mozilla/dawg/pull/13/commits/6ec96a2dbf62c73ca58b0ea82e62133a342f50a0 

<img width="1500" alt="Screenshot 2024-09-03 at 12 22 06 PM" src="https://github.com/user-attachments/assets/d4fcc161-d266-43d4-8a44-8b8018935194">